packages/components/Molecules/Card/CardEmergencyContact.vue

```diff
@@ -13,7 +13,7 @@
         :innerWrap="false"
         v-if="link"
       >
-        <rpl-icon :symbol="iconSymbol" color="primary" size="1.05"/>
+        <rpl-icon :symbol="iconSymbol" color="primary" size="1.05" :aria-labelledby="id"/>
```

Comment:
In this case we don't want to put this on the icon I think (the external one was a special case)...here it should probably go on the anchor tag itself, meaning we also need to modify that and pass it on from here.